tests/lib/rules/jsx-boolean-value.js

```diff
@@ -28,13 +28,18 @@ var ruleTester = new RuleTester({parserOptions});
 ruleTester.run('jsx-boolean-value', rule, {
   valid: [
     {code: '<App foo />;', options: ['never']},
+    {code: '<App foo />;', options: ['always', {never: ['foo']}]},
```

Comment:
Should we also cover the `always` part of these test cases? E.g. checking that `bar={true}` is still valid when the config is `['always', {never: ['foo']}]`.